src/shims/manual/mscorlib.forwards.cs

```diff
@@ -20,6 +20,7 @@
 [assembly:System.Runtime.CompilerServices.TypeForwardedTo(typeof(System.Collections.Generic.ShortEnumEqualityComparer<>))]
 [assembly:System.Runtime.CompilerServices.TypeForwardedTo(typeof(System.Collections.Generic.LongEnumEqualityComparer<>))]
 [assembly:System.Runtime.CompilerServices.TypeForwardedTo(typeof(System.Collections.ListDictionaryInternal))]
+[assembly:System.Runtime.CompilerServices.TypeForwardedTo(typeof(System.UnitySerializationHolder))]
```

Comment:
@weshaggard are you fine with that change? You can find the type here: https://github.com/dotnet/coreclr/blob/master/src/mscorlib/shared/System/UnitySerializationHolder.cs

Reply:
And any idea why I didn't need a TypeForwardedFrom attribute on the UnitySerializationHolder? Is it because it sets the type itself with SerializationInfo.SetType(Type)?

Reply:
Yes I'm fine with this change as it is along the lines as the other ones here. As for the TypeForwardedFrom question I'm not sure I would expect if you serialize on core it would end up with System.Private.CoreLib as the assembly it is from, is that not what is happening?

Reply:
That's what's so interesting about it. It doesn't contain the QualifiedAssemblyName, so it's the optimized version without the attribute:

����ÿÿÿÿ����������System.UnitySerializationHolder���Data	UnityType
AssemblyName�
�������

Reply:
Interesting I don't know why that is. Your the expert here now :) My only guess is that the binary serializer omits assembly names for the core assembly? Not sure though. It is probably worth debugging though the code to understand further.

Reply:
I have a guess but need to verify it. Thanks Wes :)